### PR TITLE
Minor fixes

### LINKED ISSUE #24: AttributeError: 'NoneType' object
**OS details**

```bash
  Operating System: CentOS Linux 7 (Core)
  Kernel: Linux 3.10.0-1127.13.1.el7.x86_64
  Architecture: x86-64
```

When parameter `attach_message_primary = True` in `inputs.conf` the following error is thrown:
```bash
Traceback (most recent call last):
  File "/opt/splunk/etc/apps/TA-mailclient/bin/mail.py", line 381, in stream_events
    self._stream_events(inputs, ew)
  File "/opt/splunk/etc/apps/TA-mailclient/bin/mail.py", line 438, in _stream_events
    self.stream_imap_emails()
  File "/opt/splunk/etc/apps/TA-mailclient/bin/mail.py", line 289, in stream_imap_emails
    self.attach_message_primary,
  File "/opt/splunk/etc/apps/TA-mailclient/bin/../lib/file_parser/email_mime.py", line 45, in parse_email
    mailheaders = Parser().parsestr(message.as_string(), True)
AttributeError: 'NoneType' object has no attribute 'as_string'
```

### PR DESCRIPTION
**Pull request by edro15**
Fixes #24, raised when:
* emails without a primary email attached had their body messages overwritten by `None`, 
* emails have a primary attachment with content-type different than the ones verified in the if-clause. 

Similarly fixed `NoneType` error occurring when parameter `maintain_rfc = True`.

### CODE REVIEW

File: lib/file_parser/email_mime.py [hunk 1]
```diff
@@ -114,6 +116,8 @@ def maintain_rfc_parse(message):
     :return: Returns a email message formatted as a string
       :rtype: str
     """
+    if message is None:
+        return "".as_string()
```
Comment (seunomosowon) [inline lib/file_parser/email_mime.py:120]: I think this might not work

Comment (edro15) [reply lib/file_parser/email_mime.py]: Apologies. You're absolutely right. Fixed that with [69bc05f](https://github.com/seunomosowon/TA-mailclient/pull/25/commits/69bc05f4f03c279334c06aea1fca46ed484d9f96). 

Basically, if you have for some reason a raw email message beginning with `\n` and `maintain_rfc` parameter set to `True`, you'd end up having a `None` object passed to `maintain_rfc_parse()` and raising 
```bash
AttributeError: 'NoneType' object has no attribute 'is_multipart'
```

Change offered in [69bc05f](https://github.com/seunomosowon/TA-mailclient/pull/25/commits/69bc05f4f03c279334c06aea1fca46ed484d9f96) will help preventing this error to occur. See also [here](https://stackoverflow.com/questions/17873091/python-msg-email-message-from-stringaaa-values-are-returning-none).

**Comment by edro15** [comment]
Hi @seunomosowon , did you have some time to check this PR out? 
Please let me know if you have more doubts or concerns about it. Thanks.

**Comment by edro15** [comment]
Hey @seunomosowon, I noticed you integrated proposed changes into the latest release of the app. Great, but next time please consider merging contributors' work when available. Besides, you may want to mark issue #24 as fixed too...

**Comment by seunomosowon** [comment]
Apologies, I think the CICD tests were failing in it so I had to rewrite some of it and I included your contributions. Thanks a lot for finding this and also contributing!

**Pull request closed without merging**